File: diemBFT/src/main/java/diem/model/QuorumCertificate.java
package diem.model;

import lombok.AllArgsConstructor;

@AllArgsConstructor
public class QuorumCertificate {

  public VoteInfo voteInfo;
  public LedgerCommitInfo ledgerCommitInfo;
  public Signature signatures;
  public Author author;
  public Signature authorSignature;
  // TODO Check BlockId not present in the code
  public int blockId;

  public QuorumCertificate(VoteInfo voteInfo, LedgerCommitInfo ledgerCommitInfo) {
  }
}

File: diemBFT/model/quorum_certificate.py
class QC:
    def __init__(self, vote_info, ledger_commit_info, signatures, author, author_signature):
        self.vote_info = vote_info
        self.ledger_commit_info = ledger_commit_info
        self.signatures = signatures
        self.author = author
        self.author_signature = author_signature
